### PR TITLE
[W13B01] added very basic output tests

### PR DESCRIPTION
**Pull request by greenbowser**
Added a simple test to compare the output generated by the code to be tested to sample output

### CODE REVIEW

File: w13b01/test/pgdp/game/BasicGameTest.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+        PrintStream t = new PrintStream(actual);
+        System.setOut(t);
+        ByteArrayInputStream in = new ByteArrayInputStream("0\n0\n".getBytes());
+        System.setIn(in);
+        PinguGame c = new PinguGame(true, 420, 0);
+        c.play();
+
+        String expectedString = Files.readString(expected.toPath());
+        String actualString = Files.readString(actual.toPath());
```
Comment (nico-lintn) [inline w13b01/test/pgdp/game/BasicGameTest.java:30]:
```suggestion
        String actualString = Files.readString(actual.toPath()).replaceAll("\r", "");
```

File: w13b01/test/pgdp/game/BasicGameTest.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+        PrintStream t = new PrintStream(actual);
+        System.setOut(t);
+        ByteArrayInputStream in = new ByteArrayInputStream("0\n1\n0\n1\n0\n1\n0\n1\n0\n0\n".getBytes());
+        System.setIn(in);
+        PinguGame c = new PinguGame(true, 420, 0);
+        c.play();
+
+        String expectedString = Files.readString(expected.toPath());
+        String actualString = Files.readString(actual.toPath());
```
Comment (nico-lintn) [inline w13b01/test/pgdp/game/BasicGameTest.java:49]:
```suggestion
        String actualString = Files.readString(actual.toPath().replaceAll("\r", ""));
```

File: w13b01/test/pgdp/game/BasicGameTest.java [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+        PrintStream t = new PrintStream(actual);
+        System.setOut(t);
+        ByteArrayInputStream in = new ByteArrayInputStream("0\n1\n0\n1\n0\n1\n0\n1\n0\n0\n".getBytes());
+        System.setIn(in);
+        PinguGame c = new PinguGame(true, 42069, 0);
+        c.play();
+
+        String expectedString = Files.readString(expected.toPath());
+        String actualString = Files.readString(actual.toPath());
```
Comment (nico-lintn) [inline w13b01/test/pgdp/game/BasicGameTest.java:68]:
```suggestion
        String actualString = Files.readString(actual.toPath()).replaceAll("\r", "");
```

**Review by nico-lintn — CHANGES_REQUESTED** [review]
As windows uses ```\r\n``` as a line seperator (and the expected files only contain ```\n```), every occurence of ```\r``` has to be removed from the actual output.
Without this change ```Assertions.assertEquals()``` will always assert ```false``` on windows.

**Comment by nilsreichardt** [comment]
can't merge because of a merge conflict

**Pull request closed without merging**